packages/integration-tests/src/entities/Author.ts

```diff
@@ -122,14 +123,19 @@ config.addRule("books", (a) => {
   }
 });
 
-// Example of rule that won't get run unless field is set
+// Example of rule that is always run even if the field is not set
```

Comment:
This behavior changed in the main field-level validation PR, but comment wasn't updated